examples/csi-config-map-sample.yaml

```diff
@@ -39,5 +48,20 @@ data:
         }
       }
     ]
+  cluster-mapping.json: |-
+    [{
+      "clusterIDMapping": {
+      "clusterID on site1: "clusterID on site2"
```

Comment:
indention looks a little off

Reply:
Yes, Thanks for catching this one. Fixed now.